packages/teuchos/parameterlist/src/Teuchos_StandardParameterEntryValidators.cpp

```diff
@@ -294,6 +296,24 @@ int AnyNumberParameterEntryValidator::getInt(
   return 0; // Will never get here!
 }
 
+long long AnyNumberParameterEntryValidator::getLongLong(
```

Comment:
These `get${TYPE}` methods seem to want a common implementation -- they all do the same thing, except for casting the return value to `${TYPE}` via `Teuchos::as`.